store/src/components/kiosk/screens/TableSelectionScreen.js

```diff
@@ -139,9 +139,11 @@ export const TableSelectionScreen = props => {
                         ></path>
                      </svg>
                   </button>
-                  <div tw="w-full text-center text-[56px] text-[#303030] font-bold">
-                     Select table
-                  </div>
+                  { config.selectTableNumberScreenSettings.showTextOnHeader.value && 
+                     <div tw="w-full text-center text-[56px] text-[#303030] font-bold">
+                        {(config.selectTableNumberScreenSettings.textOnHeader.value || "Select Table")}
```

Comment:
{(config.selectTableNumberScreenSettings.textOnHeader.value || "Select Table")}
```suggestion
                        {(config?.selectTableNumberScreenSettings?.textOnHeader?.value || "Select Table")}
```